### ISSUE TITLE
Teamplay reports just for fun, reputation system, toxic people, and ban chat.

### ISSUE THREAD

**Issue by Battal5** [ValveSoftware/Dota2-Gameplay#10244]
### Description

      While playing different modes (AP SD, Turbo and others, not ranked), there is a problem. If i join game with team of 4 (skip team of 4 is not working in non ranked game), i gain 100% LP. The reason of this 50% just for fun, because they just can do it, so why not? And other 50% because team of 4 will never play with 1 other player same quality as 5 random people play with.

       As a result there is ZERO possibility to get out of low reputation status. Because each 4-6 of turbo games i get 1 time team of 4, with 100% LP after. So even if i will grab 50% of likes in 4-6 games, i would get 6-14 likes what is approx +600 to +1400 reputation, and after i got LP i get -1000 and extra -100 for each report was made. Last 4 month i stay at low reputation. But each time i grab 1500-2000 reputation, i got 1 or 2 team of 4 players, got LP and same will happen again and again. Last time i was in low reputation, it was much easy to get back to 10k reputation. Because there was no chat ban.

     Now there is voice and chat ban, if you got less than 3k reputation. There was only 1 time, i had got 3000+ reputation at this 4 months. I always play as a gentelmen, and do my best to communicate with team if team could be not flaming. If i hear flaming, i just mute them manually. In my opinion automatic chat ban in low reputation is a 100% disaster. While i play with people who flaming all game long, and get no penalty for that. (in older times, there was working reports with ban chat). Now there is NO ban chat while people are reported for all game flaming. 

This problem could be solved easy. 
1st option is refuse to report team of 4 other lonely team player. 
2nd option is double check 4 reports from team of 4, as a 50% fake opportunity.
3rd option is move player to Low Priority games, but without remove his reputation.

And please remove this automatic ban chat with lower 3k reputation. This is unfair. (return ban chat system, when players report flaming persons, it was rly good times.)

P.S. if you will give penalty to people who use pause more than 2 times in 1 game, dota will be less toxic. (now dota is intoxicated with pause abuse bulling people)

**Comment by eric1294** [comment]
The behavior score system just needs to go.

**Comment by Battal5** [comment]
no, behavior score system is a good idea. bad guys plays with same bad guys and enjoy their play. but if you somehow go to the bottom of the score (for example try to play new hero, and get mass reported), it is almost impossible to go back to the top now.

**Comment by eric1294** [comment]
Bad guys play with bad guys in low priority where they belong. The behavior score system makes it take too long to find matches.
